### PR TITLE
chore: Update bug report template

### PR DESCRIPTION
**Pull request by naman653**
Changes: Bug report template was not correctly displaying the device info table.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -20,8 +20,9 @@ about: Create a bug report to help us improve
 <!-- If applicable, add screenshots to help explain your problem. -->
 
 **Smartphone Info:**
-|               | |
-|---------------|-|
+
+|               |   |
```
Comment (yashk2000) [inline .github/ISSUE_TEMPLATE/bug_report.md]: unnecessary change.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -20,8 +20,9 @@ about: Create a bug report to help us improve
...
+|---------------|---|
```
Comment (yashk2000) [inline .github/ISSUE_TEMPLATE/bug_report.md]: unnecessary change. Only the empty line is required.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (naman653) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I know, but these are in accordance with official guidelines of github. You can find it here https://help.github.com/en/articles/organizing-information-with-tables
